1host.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+	port: 3000,
+	modules: [
+		{
+			module: require("./index.js"),
```

Comment:
Codacy has a fix for the issue: [Do not reference the index file directly..](https://app.codacy.com/gh/onedotprojects/scratchCommentAuth/pullRequest?prid=7364619)
```suggestion
			module: require('.'),
```